third_party/glad/CMakeLists.txt

```diff
@@ -8,6 +8,11 @@ target_include_directories(
     "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>"
     "$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>"
 )
+set_target_properties(
+    glad
+    PROPERTIES
+    POSITION_INDEPENDENT_CODE ON
+)
```

Comment:
this adds -fPIC compilation option which should be always added for static libraries linked do dynamic ones (I was getting 'error: lib.a: relocation R_X86_64_PC32 against symbol `...' can not be used when making a shared object' errors without it)